feg/protos/s6a_proxy.proto

```diff
@@ -140,6 +140,8 @@ message UpdateLocationRequest {
     bool skip_subscriber_data = 3; // bit 2
     // Send Cancel Location to other EPCs serving the UE
     bool initial_attach = 4; // bit 5
+    //Send Dual Registration 5G Indicator
+     bool dual_regis_5g_ind = 5; // bit 8
```

Comment:
minor nit but spacing looks a bit off here.

Reply:
We normally don't abbreviate on protos. Let's use the full name. 

Reason is that protons are used across different services and its field name can help to provide context. 

```
bool send_dual_registration_5g_indicator
```